src/main/kotlin/ru/buhinder/alcopartyservice/AlcopartyServiceApplication.kt

```diff
@@ -5,9 +5,9 @@ import org.springframework.boot.context.properties.ConfigurationPropertiesScan
 import org.springframework.boot.context.properties.EnableConfigurationProperties
 import org.springframework.boot.runApplication
 
+@SpringBootApplication
 @ConfigurationPropertiesScan
 @EnableConfigurationProperties
-@SpringBootApplication
 class AlcopartyServiceApplication
```

Comment:
would be good to have task number format in the conmmit message like this [BUH-00]